opentreemap/treemap/audit.py

```diff
@@ -743,12 +743,25 @@ def user_can_delete(self, user):
         field permissions on a model.
         """
 
-        is_admin = user.get_role(self.instance).name == Role.ADMINISTRATOR
+        is_admin = user.get_instance_user(self.instance).admin
```

Comment:
It has never been great that we match in the name of the Administrator role, but the `admin` flag on `InstanceUser` is used to control access to the management pages. If we couple these together, we cannot create a user that can delete map feature data without also letting them delete custom fields, etc. This may be ok, but there should be some discussion about this change if there hasn't been already.